apps/antalmanac/src/components/RightPane/CoursePane/SearchForm/SearchForm.tsx

```diff
@@ -50,14 +50,29 @@ const styles: Styles<Theme, object> = {
 const SearchForm = (props: { classes: ClassNameMap; toggleSearch: () => void }) => {
     const { classes, toggleSearch } = props;
     const { manualSearchEnabled, toggleManualSearch } = useCoursePaneStore();
+    const [helpBoxVisibility, setHelpBoxVisibility] = useState(true);
 
     const onFormSubmit = (event: FormEvent) => {
         event.preventDefault();
         toggleSearch();
     };
 
     const currentMonthIndex = new Date().getMonth(); // 0=Jan
-    const activeMonthIndices = [false, false, false, false, false, false, false, false, true, true, false, false];
+    // Active months: February/March for Spring planning, May/June for Fall planning, August for Summer planning,
+    // and November/December for Winter planning
+    const activeMonthIndices = [false, true, true, false, true, true, false, true, false, false, true, true];
```

Comment:
```suggestion
    const activeMonthIndices = [false, true, true, false, true, true, true, true, false, false, true, true];
```

Add July to active months for SPOPs